beacon-chain/rpc/eth/beacon/pool.go

```diff
@@ -367,17 +367,16 @@ func (bs *Server) SubmitSignedBLSToExecutionChanges(ctx context.Context, req *et
 }
 
 func (bs *Server) broadcastBLSBatch(ctx context.Context, ptr *[]*ethpbalpha.SignedBLSToExecutionChange) {
```

Comment:
Can you add a comment to imply `ptr` will get mutated as they broadcast out?

Reply:
and I'm not sure of `ptr` is the best name, maybe consider `messages`

Reply:
being pedantic, but the value ptr references will be mutated as its gets broadcast out